### PR TITLE
ENG-497 fix plural crypto recover command

### PR DESCRIPTION
**Pull request by zreigz**
## Summary
This PR fixes the` plural crypto recover` command. It will also fix the broken `~/.plural/key`

## Checklist

- [ ] If required, I have updated the Plural documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] I have added a meaningful title and summary to convey the impact of this PR to a user.
- [ ] I have added relevant labels to this PR to help with categorization for release notes.

### CODE REVIEW

File: cmd/plural/crypto.go [hunk 1]
```diff
@@ -371,8 +372,14 @@ func handleRecover(c *cli.Context) error {
 	if !ok {
 		return fmt.Errorf("could not find `key` in console-conf secret")
 	}
+	keyValue := string(key)
```
Comment (DavidSpek) [inline cmd/plural/crypto.go:375]: Wouldn't it be possible to reuse the AESKey struct and related functions here?

```go
type AESKey struct {
	Key string
}
```

They're in `pkg/crypto/key.go`.

Comment (zreigz) [reply cmd/plural/crypto.go:375]: good point let me check this

Comment (zreigz) [reply cmd/plural/crypto.go]: works fine, thanks

**Review by DavidSpek — APPROVED** [review]
Thanks for the quick response for this issue!

**Pull request merged**